### DISCUSSION TITLE
New Project Roadmap not showing the milestone due date

### DISCUSSION THREAD

**Discussion by kjetildk** [community/community#46023] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi,

When using the Project Roadmap view and include grouping on milestone the due date (if used) should be included:

1. Due date should show the length of the milestone, not the issues connected to it. 
3. Due date could also be shown on the right side of the milestone name in the left column 

Regards
Kjetil

**Comment by minware-dan** [comment]
Hey @kjetildk -

Great recommendations. Do you have experience with any other roadmap services? 

This is something we handle pretty well in minware - the project due date shows up as a milestone above the milestone gantt charts and the tool projects future work completion based on historical velocity. You can clearly see when a project is expected to wrap up after the due date.

**Comment by rileybroughten** [comment]
Hello @kjetildk - you can now [display vertical markers for milestone due dates](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-roadmap-layout#setting-vertical-markers) associated with your project items! Give it a try and let us know your thoughts 👍

**Reply by kjetildk** [reply]
Wow! Great News! We will definitely try it out 😃
